tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -951,6 +953,6 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args, python_api_us
 
         send_conversion_result('fail')
         if python_api_used:
-            raise e#.with_traceback(None)
+            raise e.with_traceback(None)
```

Comment:
why is this changed?

Reply:
It looks like we have the wrong change on master, so I uncommented this code. If we intentionally commented it I suggest to change to `raise e` and remove the commented code.